code/modules/mining/orm_vr/equipment_vendor.dm

```diff
@@ -41,6 +41,7 @@
 		new /datum/data/mining_equipment("Industrial Hardsuit - Material Scanner",	/obj/item/rig_module/vision/material,					500),
 		new /datum/data/mining_equipment("Industrial Hardsuit - Maneuvering Jets",	/obj/item/rig_module/maneuvering_jets,					1250),
 		new /datum/data/mining_equipment("Diamond Pickaxe",				/obj/item/weapon/pickaxe/diamond,									2000),
+		new /datum/data/mining_equipment("Diamond Drill",				/obj/item/weapon/pickaxe/diamonddrill,								12500),
```

Comment:
Add a `//CITADEL CHANGE - adds diamond drills to the equipment vendor`

Reply:
done :^)